packages/lib/src/common/variables.ts

```diff
@@ -366,6 +366,17 @@ export const componentTokens = {
     focusColor: CoreTokens.color_blue_600,
     scrollBarThumbColor: CoreTokens.color_grey_700,
     scrollBarTrackColor: CoreTokens.color_grey_300,
+    // action icon tokens
```

Comment:
Shouldn't all the tokens contain the `actionIcon` keyword? (Instead of just action for the last 5)